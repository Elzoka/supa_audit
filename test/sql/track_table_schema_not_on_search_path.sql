begin;
    -- Check escaping rules for schemas not on the search path
    -- and tables containing capital letters

    create schema xyz;

    create table xyz."Members"(
        id int primary key,
        name text not null
    );

    select audit.enable_tracking('xyz."Members"');
    select audit.disable_tracking('xyz."Members"');

rollback;
