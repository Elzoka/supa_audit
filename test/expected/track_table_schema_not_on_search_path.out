begin;
    -- Check escaping rules for schemas not on the search path
    -- and tables containing capital letters
    create schema xyz;
    create table xyz."Members"(
        id int primary key,
        name text not null
    );
    select audit.enable_tracking('xyz."Members"');
 enable_tracking 
-----------------
 
(1 row)

    select audit.disable_tracking('xyz."Members"');
 disable_tracking 
------------------
 
(1 row)

rollback;
